wandb/sdk/wandb_init.py

```diff
@@ -72,13 +73,32 @@ def _maybe_mp_process(backend: Backend) -> bool:
     return False
 
 
+def _handle_launch_config(settings: "Settings") -> Dict[str, Any]:
+    launch_run_config = {}
+    if settings.launch and (os.environ.get("WANDB_CONFIG") is not None):
+        try:
+            launch_run_config = json.loads(os.environ.get("WANDB_CONFIG", "{}"))
+        except (ValueError, SyntaxError):
+            wandb.termwarn("Malformed WANDB_CONFIG, using original config")
```

Comment:
this logic seems to be repeated here and in `wandb_run`

Reply:
The environ is an actual config? ( i don't see it in `env.py`)

Reply:
I was under the impression that we only added things to env.py when they affected the settings object?

Reply:
oh yeah... i was just trying to understand what exactly this env variable means...

Reply:
Oh, if this environment variable exists and the WANDB_LAUNCH environment variable is true, we the run config will be updated with the value of this environment variable